v2/bundle/jwtbundle/set.go

```diff
@@ -61,7 +61,8 @@ func (s *Set) GetJWTBundleForTrustDomain(trustDomain spiffeid.TrustDomain) (*Bun
 	s.mtx.RLock()
 	defer s.mtx.RUnlock()
 
-	if !s.Has(trustDomain) {
+	_, ok := s.bundles[trustDomain]
```

Comment:
how would you feel about capturing the bundle you looked up and returning that below instead of redoing the lookup?
```suggestion
	bundle, ok := s.bundles[trustDomain]
```

Reply:
Sure, didn't notice that 🤦‍♂